tests/libtest.sh

```diff
@@ -76,6 +87,23 @@ cleanup() {
   wipe_signatures "$devs"
 }
 
+cleanup_container_root_volume() {
```

Comment:
This function feels very specific. It probably is better in the test itself and not part of library.

Or, break this down into smaller pieces. That is have a helper function to cleanup mount unit file in libtest.sh and logic to remove lvm keep it in test itself.